content/blog/qmstr-sprint-4.3-report.md

```diff
@@ -6,10 +6,60 @@ categories = ["qmstr"]
 banner = "img/blog/qmstr-sprint-4.3-title.jpg"
 +++
 
-And some features.
+The QMSTR sprint 4.2 and 4.3 (this report covers both) are part of our
+current efforts to merge the ongoing QMSTR development and the FASTEN
+project. A key necessity for this was documentation. Everybody loves
+to write documentation, especially developers. Right? :-) To make it
+even more fun, we work on integrating end user and administrator
+documentation with the project code and perform automated integration
+test automatically. We hope that this approach may lead to
+documentation that works and is reliable. Even though this effort is
+still ongoing, some results are already visible, like the new project
+documentation on the main web site. Some feature development primarily
+on the node management command line interface is also in progress and
```

Comment:
I would suggest to add some commas in this sentence as it can be a bit confusing.